benchmarks/benchmark_serving.py

```diff
@@ -792,16 +819,22 @@ def main(args: argparse.Namespace):
   )
   parser.add_argument(
       "--run-eval",
-      type=bool,
+      type=str2bool,
       default=False,
       help="Whether to run evaluation script on the saved outputs",
   )
   parser.add_argument(
       "--warmup-first",
-      type=bool,
+      type=str2bool,
       default=False,
       help="Whether to send warmup req first",
```

Comment:
We should combine with "--warmup-first" option and instead add new option  "--warmup-mode" with possible values = [None, sampled, full]. We already have so many options, need to stop the bloat :).

Reply:
Done